ci/code_checks.sh

```diff
@@ -81,15 +81,12 @@ if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
         -i "pandas.CategoricalIndex.ordered SA01" \
         -i "pandas.DataFrame.__dataframe__ SA01" \
         -i "pandas.DataFrame.__iter__ SA01" \
-        -i "pandas.DataFrame.assign SA01" \
         -i "pandas.DataFrame.at_time PR01" \
         -i "pandas.DataFrame.axes SA01" \
-        -i "pandas.DataFrame.bfill SA01" \
         -i "pandas.DataFrame.columns SA01" \
         -i "pandas.DataFrame.copy SA01" \
         -i "pandas.DataFrame.droplevel SA01" \
         -i "pandas.DataFrame.dtypes SA01" \
-        -i "pandas.DataFrame.ffill SA01" \
```

Comment:
There is also a Series.bfill/ffill a bit below that has to be removed as well

Reply:
Thank you for pointing that out!